packages/p2p/src/components/orders/chat/chat-message-receipt.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@deriv/components';
+import ChatMessage from 'Utils/chat-message';
+
+const ChatMessageReceipt = ({ chat_channel, message, sendbird_user_id }) => {
+    let icon_name;
+
+    if (message.status === ChatMessage.STATUS_PENDING) {
+        icon_name = 'IcMessagePending';
+    } else if (message.status === ChatMessage.STATUS_ERRORED) {
+        icon_name = 'IcMessageErrored';
+    } else {
+        const channel_user_ids = Object.keys(chat_channel.cachedReadReceiptStatus);
+        const other_sendbird_user_id = channel_user_ids.find(user_id => user_id !== sendbird_user_id);
+
+        // User's last read timestamp is larger than or equal to this message's createdAt.
+        if (chat_channel.cachedReadReceiptStatus[other_sendbird_user_id] >= message.created_at) {
+            icon_name = 'IcMessageSeen';
+        } else {
+            icon_name = 'IcMessageDelivered';
+        }
+    }
+
+    return <Icon className='order-chat__messages-item-timestamp-receipt' icon={icon_name} size={16} />;
+};
+
+ChatMessageReceipt.propTypes = {
+    active_chat_channel: PropTypes.object,
```

Comment:
`active_chat_channel` doesn't exist in this component.

Reply:
Remnants of a previous version, fixed that. 👍